llvm/tools/llc/llc.cpp

```diff
@@ -338,6 +354,35 @@ struct LLCDiagnosticHandler : public DiagnosticHandler {
   }
 };
 
+// Returns a pointer to a CAS using the CLI parameters.
```

Comment:
Some editors have special support for doxygen comments in their online help, so using /// even for static functions can be nice.

Reply:
Good catch!